<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이미지 이름:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>580</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; 이전</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>580</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>다음 &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cur_image_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="cur_image_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="bbox_check">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BBox 보기</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="cur_idx">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="image_num">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>총 이미지 수:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cur_idx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>search(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>708</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbox_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>show_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>714</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>prev_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>next_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>889</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_image()</slot>
  <slot>modify_image()</slot>
  <slot>view_bbox(bool)</slot>
  <slot>search(int)</slot>
  <slot>next_class()</slot>
  <slot>prev_class()</slot>
  <slot>next_image()</slot>
  <slot>prev_image()</slot>
  <slot>new_image()</slot>
  <slot>show_image()</slot>
 </slots>
</ui>
